core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -446,7 +446,7 @@ final class Stream[+F[_], +O] private[fs2] (private[fs2] val underlying: Pull[F,
   /** Outputs chunks of size `n`.
     *
     * Chunks from the source stream are split as necessary.
-    * If `allowFewer` is true, the last chunk that is emitted may have less than `n` elements.
+    * If `allowFewer` is true, the last chunk that is emitted, upon completion of the stream, may have less than `n` elements.
```

Comment:
A small mistake that was already there.

```suggestion
    * If `allowFewer` is true, the last chunk that is emitted, upon completion of the stream, may have fewer than `n` elements.
```

Reply:
Ah yes, thank you!